types/jasmine/index.d.ts

```diff
@@ -382,6 +407,16 @@ declare namespace jasmine {
         equals(a: any, b: any, customTesters?: ReadonlyArray<CustomEqualityTester>): boolean;
         contains<T>(haystack: ArrayLike<T> | string, needle: any, customTesters?: ReadonlyArray<CustomEqualityTester>): boolean;
         buildFailureMessage(matcherName: string, isNot: boolean, actual: any, ...expected: any[]): string;
+
+        /**
+         * Formats a value for use in matcher failure messages and similar
+         * contexts, taking into account the current set of custom value
+         * formatters.
+         * @since 3.6.0
+         * @param value - The value to pretty-print
+         * @return - The pretty-printed value
```

Comment:
Drop the - before The?

Reply:
Thanks for the review :D 

> This looks ok but I'd prefer any over unknown to be consistent with current types.

Agreed with the consistency argument. In general, we should move to using more unknown, especially in defining function types to force type check. Leaving as is if you don't have strong preference.

> Drop the - before The?

Done. Updated for the whole file.